src/shogun/lib/SGSparseMatrix.h

```diff
@@ -105,12 +105,12 @@ template <class T> class SGSparseMatrix : public SGReferencedData
 		 */
 		inline const T operator()(index_t i_row, index_t i_col) const
 		{
-			REQUIRE(i_row>=0, "index %d negative!\n", i_row);
-			REQUIRE(i_col>=0, "index %d negative!\n", i_col);
-			REQUIRE(i_row<num_features, "index should be less than %d, %d provided!\n",
-				num_features, i_row);
-			REQUIRE(i_col<num_vectors, "index should be less than %d, %d provided!\n",
-				num_vectors, i_col);
+			REQUIRE(i_row>=0, "Index %d negative!\n", i_row);
```

Comment:
Sorry I did not spot this earlier, but "Index" is ambiguous. Please use row or column index